lib/interfaces/uncaught.js

```diff
@@ -54,6 +54,7 @@ function handlerSetup(client, config) {
 
     errorHandlerRouter(err, em);
     client.sendError(em, handleProcessExit);
+    setTimeout(handleProcessExit, 2000);
```

Comment:
@matthewloring  This would be difficult to do/gain value out of since we already test the transport layer and sending messages seperatley while to test this we would need to either rely on a timer and attempt to check for a new error (inaccurate and flaky since we can't query for specific errors) or attempt to monkey path the library itself for testing so that we can do the same integration test we already to with the transport layer with the only difference being the call from the uncaught exception event listener. Essentially these components are individually tested and trying to insert instrumentation to test the two working together would require changing the way the components work so that we would be able to test them.